src/components/KioskLayout.tsx

```
@@ -0,0 +1,801 @@
+              <div className={styles.couponField}>
+                <div className={styles.couponInput}>
+                  <Tag size={18} className={styles.couponIcon} />
+                  <input 
+                    type="text" 
+                    value={couponCode}
+                    onChange={(e) => setCouponCode(e.target.value)}
+                    placeholder="Ingrese código de cupón"
+                    className={styles.couponCodeInput}
+                    autoFocus
+                  />
+                </div>
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)